src/components/gltf-model-plus.js

```diff
@@ -200,7 +200,7 @@ function cachedLoadGLTF(src, basePath, contentType, preferredTechnique, onProgre
       try {
         let gltfUrl = src;
         let onLoad = resolve;
-        if (contentType === "model/gltf+zip") {
+        if (contentType === "model/gltf+zip" || contentType === "application/x-zip-compressed") {
```

Comment:
I guess this is ok, its a bit odd though since currently the zip loading code assumes sketchfab's format (namely that there is a scene.gtlf in the root of the zip), so its not technically generally applicable to all zips.